packages/tooling/write-dockerfile.ts

```diff
@@ -0,0 +1,39 @@
+import { execSync } from "child_process";
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import { join } from "path";
+
+const workspaceRoot = join(__dirname, "../..");
+const dockerFilePath = join(workspaceRoot, "web-server.Dockerfile");
+
+console.log(`writing ${dockerFilePath} ...`);
+
+const stdout = execSync(
+  `lerna list --include-dependencies --json --all --loglevel silent --scope @dzcode.io/web-server`,
+);
+const dependencies = JSON.parse(stdout.toString()) as Array<{ name: string; location: string }>;
```

Comment:
```suggestion
const dependencies = JSON.parse(stdout.toString()) satisfies Array<{ name: string; location: string }>;
```
Just to add compile time safety